### ISSUE TITLE
Build gradle stuck on app:compileDebugKotlin

### ISSUE THREAD

**Issue by slikasgiedrius** [android/compose-samples#335]
For some reason after adding all jetpack compose dependencies building to device works fine, but once I add **compose true** the building process is running forever on app:compileDebugKotlin and no result.. Anyone can help?

**Comment by slikasgiedrius** [comment]
Solved by adding 

 implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
    implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"

**Comment by slikasgiedrius** [comment]
Nah, it's still there

**Comment by madushans** [comment]
Ran into this with alpha09.
Try removing other plugins.

In my case I had to remove `kotlinx-serialization` and `kotlin-android-extensions` for it to work.

`kotlinx-serialization` [does not support being on the same module as compose](https://github.com/Kotlin/kotlinx.serialization/issues/968) but likely will be 'fixed' [around compose Beta release](https://github.com/Kotlin/kotlinx.serialization/issues/968#issuecomment-707046960).

and `kotlin-android-extensions` was used for synthetic extensions and is [on the way out](https://developer.android.com/topic/libraries/view-binding/migration) in favor of view-bindings.

Unfortunately if you're using either of the above in an existing app and still want to add Compose, it appears they atleast cannot be in the same module and may require some refactoring to move them out before adding compose stuff, or may have to create a new module to house Compose things, depending on what's easier for your app.

If removing above still doesn't solve it, try running [process monitor (Windows)](https://docs.microsoft.com/en-us/sysinternals/downloads/procmon) or something similar and see which folders gradle (java.exe) accesses last, just before getting stuck. This may point to the plugin in question.
